config/config.go

```diff
@@ -99,6 +99,10 @@ type Configuration struct {
 	// BidderInfos supports adapter overrides in extra configs like pbs.json, pbs.yaml, etc.
 	// Refers to main.go `configFileName` constant
 	BidderInfos BidderInfos `mapstructure:"adapters"`
+	// Modules mapping provides module specific configuration
+	Modules map[string]interface{} `mapstructure:"modules"`
```

Comment:
Yes, you got it right.

If we refuse to use the `interface{}` type here, module developers will need to edit the modules config file located in the root `config/` directory, which we are trying to avoid.